.github/build-scripts/patch-wheel.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin
+set -e
+set -x
+
+WHL=ucx_py-0.28.0a0+37.ge1f0547-cp38-cp38-manylinux_2_31_x86_64.whl
+
+# first grab the auditwheel hashes for libuc{tms}
+LIBUCM=`unzip -l $WHL | grep -E "libucm-.*.so\.0\.0\.0" | awk '{ printf "%s\n",$4 }' | cut -d '-' -f 2 | cut -d '.' -f 1`
+LIBUCT=`unzip -l $WHL | grep -E "libuct-.*.so\.0\.0\.0" | awk '{ printf "%s\n",$4 }' | cut -d '-' -f 2 | cut -d '.' -f 1`
+LIBUCS=`unzip -l $WHL | grep -E "libucs-.*.so\.0\.0\.0" | awk '{ printf "%s\n",$4 }' | cut -d '-' -f 2 | cut -d '.' -f 1`
```

Comment:
```suggestion
LIBUCM=$(unzip -l $WHL | awk 'match($4, /libucm-[^\.]+\./) { print substr($4, RSTART) }')
LIBUCT=$(unzip -l $WHL | awk 'match($4, /libuct-[^\.]+\./) { print substr($4, RSTART) }')
LIBUCS=$(unzip -l $WHL | awk 'match($4, /libucs-[^\.]+\./) { print substr($4, RSTART) }')
```

Let's just grab the entire library name.

Reply:
+1